InnerEye/ML/deep_learning_config.py

```diff
@@ -647,11 +650,24 @@ def use_gpu(self, value: bool) -> None:
     @property
     def use_data_parallel(self) -> bool:
         """
-        Data parallel is used if GPUs are usable and the number of CUDA devices are greater than 1.
+        Data parallel is used if GPUs are usable and the number of CUDA devices are greater than 1 and
+        DistributedDataParallel is False (i.e. is_windows is True)
```

Comment:
what's that thing about is_windows?

Reply:
The previous pytorch version's implementation of ddp didn't support windows. Under the new version it might - I haven't tried it on a windows machine yet